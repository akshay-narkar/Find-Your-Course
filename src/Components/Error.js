export default function Error() {
  return (
    <>
      <h1>Oops! You have taken the wrong route! </h1>
    </>
  );
}
