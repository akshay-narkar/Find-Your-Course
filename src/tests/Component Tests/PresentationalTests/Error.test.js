import renderer from 'react-test-renderer';
import Error from '../../../Components/Error';

it('UI is unaltered wrt first snapshot', () => {
  const elem = renderer.create(<Error />).toJSON();

  expect(elem).toMatchSnapshot();
});
