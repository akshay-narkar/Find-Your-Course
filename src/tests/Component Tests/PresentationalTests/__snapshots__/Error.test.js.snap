// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI is unaltered wrt first snapshot 1`] = `
<h1>
  Oops! You have taken the wrong route! 
</h1>
`;
